pkg/model/planpreview.go

```diff
@@ -14,10 +14,30 @@
 
 package model
 
+import "time"
+
 func (r *PlanPreviewCommandResult) FillURLs(baseURL string) {
 	r.PipedUrl = MakePipedURL(baseURL, r.PipedId)
 	for _, ar := range r.Results {
 		ar.ApplicationUrl = MakeApplicationURL(baseURL, ar.ApplicationId)
 		ar.EnvUrl = MakeEnvironmentURL(baseURL, ar.EnvId)
 	}
 }
+
+func MakeApplicationPlanPreviewResult(app Application, env *Environment) *ApplicationPlanPreviewResult {
```

Comment:
Instead, how about receiving `envName string`. Nil check for `env` can be done at where callers do.

Reply:
(This is nit)

Reply:
👍